### PR TITLE
Bearcat Rework

### PR DESCRIPTION
**Pull request by BurpleBineapple**
🆑 
rscadd: Adds a new landing only shuttle to the bearcat with enough space for 5 people.
rscadd: Adds a functioning fission generator to the Bearcat's engineering department.
rscadd: Fixes all damage to the aft of the Bearcat, allowing players to be able to fix it and fly around early into the round.
tweak: The Bearcat's generator should be enough to last the ship on low engine use even without solars now.
rscadd: A mini-armory to the bearcat with 3 mosins and 3 pistols
/🆑 

Basically, makes the Bearcat great again.
image
image
image

### CODE REVIEW

**Comment by BladeburstNINJA** [comment]
Looks great! Can you add a suit cycler as well?

**Comment by BurpleBineapple** [comment]
Changing this to a draft while I fix an issue with the map runtiming

**Comment by BurpleBineapple** [comment]
> Looks great! Can you add a suit cycler as well?

And yeah, I'll throw a suit cycler in the back of the shuttle once I've got a moment

**Comment by BurpleBineapple** [comment]
Should be ready for review once this builds

**Review by Superhats — CHANGES_REQUESTED** [review]
Looks good, beyond a map conflict.

**Review by Superhats — APPROVED** [review]
Yep.
